examples/simple-lambda-worker/lib/simple-lambda-worker-stack.ts

```
@@ -0,0 +1,73 @@
+import * as cdk from "@aws-cdk/core";
+import * as sns from "@aws-cdk/aws-sns";
+
+import { LambdaWorker } from "../../../lib";
+
+export class SimpleLambdaWorkerStack extends cdk.Stack {
+  constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    // Use AWS_PREFIX to give all resources in this sample
+    // a unique name. This is usually `development-xx` where xx are your initials.
+    // If you do not set AWS_PREFIX, when you deploy this stack, it may conflict
+    // with someone elses stack who has also not set AWS_PREFIX
+    const prefix = process.env.AWS_PREFIX
```

Maybe throw an error instead, otherwise people will have to look to find it.